deploy/olm-catalog/service-telemetry-operator/manifests/infra.watch_servicetelemetrys_crd.yaml

```diff
@@ -149,6 +149,79 @@ spec:
                       type: object
                   type: object
               type: object
+            logs:
+              description: Logs related backend configuration.
+              properties:
+                loki:
+                  description: Logs storage backend Loki
+                  properties:
+                    enabled:
+                      description: Enable Loki as a storage backend for logs
+                      type: boolean
+                    replicationFactor:
+                      description: Loki replication factor
+                      type: string
+                    size:
```

Comment:
Generally `size` is used to indicate the number of replicas. I think this might be better as `flavor` or something like that.